deployment/secure_research_desktop/packages/packages-python.yaml

```diff
@@ -24,9 +24,6 @@ packages:
     gym:
     html5lib:
     ipykernel:
-    keras:
-        "3.11": ["uninstallable"]  # requires tensorflow
-    lightgbm:
```

Comment:
Instead of removing `lightgbm` could you mark it as `uninstallable` for all Python versions and explain why?